kratos/CMakeLists.txt

```diff
@@ -143,7 +143,8 @@ target_link_libraries(Kratos KratosCore)
 set_target_properties(Kratos PROPERTIES PREFIX "")
 
 if(USE_COTIRE MATCHES ON)
-    ## KratosCore its ok
+    ## Kratos cannot be compiled with just one unity build as it takes way too much ram
+    SET(COTIRE_MAXIMUM_NUMBER_OF_UNITY_INCLUDES "-j2")
```

Comment:
i think that @roigcarlo changes this to -j8, so i think that this specific file should be left untouched. Carlos any comment?

Reply:
I added the -j2 as well becase there were problems with the output. It seems that if travis dosn't see anything for 10 min it halts, so I split that in 2